compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/PropertyReferenceLowering.kt

```diff
@@ -95,6 +99,33 @@ internal class PropertyReferenceLowering(val context: JvmBackendContext) : Class
             calculateOwner(expression.propertyContainer, this@PropertyReferenceLowering.context)
         }
 
+    private fun IrBuilderWithScope.computeSignatureString(expression: IrCallableReference): IrExpression {
+        return expression.getter?.let {
+            localPropertyIndices[it]?.let { irString("<v#$it>") }
```

Comment:
Please rename the outer `it` here to something like `getter` to fix the inspection

Reply:
Good idea, the squiggly was so small that I didn't even notice it. Thanks!